src/essreflectometry/amor/types.py

```diff
@@ -0,0 +1,30 @@
+from typing import NewType
+
+import sciline
+import scipp as sc
+
+from ..types import Run
+
+WavelengthResolution = NewType('WavelengthResolution', sc.Variable)
+AngularResolution = NewType('AngularResolution', sc.Variable)
+SampleSizeResolution = NewType('SampleSizeResolution', sc.Variable)
```

Comment:
Why AMOR-specific?

Reply:
I think the uncertainty estimates are AMOR specific.
The docs strings of the resolution providers reference the AMOR paper.